shell.nix

```diff
@@ -29,9 +29,10 @@ pkgs.mkShell {
       # only needed for older versions
       # openssl
 
-      # functional tests
+      # functional tests & linting
       python3
       python3Packages.flake8
+      python3Packages.lief
```

Comment:
I didn't know the python lief was usd/required. Do you happen to know where it's being used and for what?

Reply:
Ah, thought I got it into the commit message but it's not there.. `lief` is required by `/test/lint/lint-python.py`.